service/test/endToEnd/teamsTest.js

```diff
@@ -70,26 +77,23 @@ lab.experiment('Teams - get/list', () => {
   })
 
   lab.test('get single team', (done) => {
-    const options = utils.requestOptions({
-      method: 'GET',
-      url: '/authorization/teams/1'
-    })
-
-    server.inject(options, (response) => {
-      const result = response.result
+    teamOps.createTeam(teamData, (err, team) => {
+      expect(err).to.not.exist()
 
-      expect(response.statusCode).to.equal(200)
-      expect(result).to.equal({
-        id: 1,
-        name: 'Admins',
-        organizationId: 'WONKA',
-        description: 'Administrators of the Authorization System',
-        path: '1',
-        users: [{ id: 'AugustusId', name: 'Augustus Gloop' }],
-        policies: [{ id: 1, name: 'Director', version: '0.1' }]
+      const options = utils.requestOptions({
+        method: 'GET',
+        url: `/authorization/teams/${team.id}`
       })
 
-      done()
+      server.inject(options, (response) => {
+        const result = response.result
+
+        expect(response.statusCode).to.equal(200)
+        expect(result.id).to.equal(team.id)
+        expect(result.name).to.equal(team.name)
```

Comment:
I did not want to have the ids references here (teams id, policies ids, users ids). As long as we have input/output format validation with Joi we should be quite safe on that side